### PR TITLE
feat(DataBrowser): hide Sample Data if configured

### LINKED ISSUE #380: Hide Sample Data according section with config.json file
If VolView is loading data via URL parameter, integrators may want to hide the Sample Data section.

Add a entry in the config.json file to remove the Sample Data section from the Data Browser tab. 

**Sample config**
```
{
  "sampleData": {
    "hide": true
  },
  "labels": {
    "lesion": { "color": "#ff0000" },
    "tumor": { "color": "green" },
    "innocuous": {
      "color": "white"
    }
  }
}
```

### PR DESCRIPTION
**Pull request by PaulHax**
Kind of silly to make a new store just for this? 🤷  Simpler way?

Closes #380

### CODE REVIEW

File: src/io/import/processors/handleConfig.ts [hunk 1]
```diff
@@ -3,36 +3,46 @@ import { z } from 'zod';
...
   })
 );
 
-const Config = z.object({
-  labels: z.record(Label).or(z.null()).optional(),
-  rulerLabels: z.record(RulerLabel).or(z.null()).optional(),
-  rectangleLabels: z.record(RectangleLabel).or(z.null()).optional(),
+const sampleData = z
+  .object({
+    hide: z.boolean().optional(),
```
Comment (floryst) [inline src/io/import/processors/handleConfig.ts:25]: I would recommend making this similar in structure to the store:

```
dataBrowser: {
  hideSampleData: boolean
}
```

**Review by floryst — COMMENTED** [review]
What do you think about adding an empty state for the sidebar if there are no panels? Something that says "No data loaded". There should be a CSS way to detect this.

I'm okay with the extra store, since it keeps with the idea that the UI should be driven by the stores.

**Comment by PaulHax** [comment]
Added the "No data loaded" message.

File: src/io/import/processors/handleConfig.ts (hunk 1 above)
Comment (PaulHax) [reply src/io/import/processors/handleConfig.ts]: Fixed.

**Comment by floryst** [comment]
LGTM!

**Pull request merged**